style/forms/ToggleButton.scss

```diff
@@ -0,0 +1,46 @@
+$slider-transition: 0.4s;
+
+.ds-form-control-toggle {
+  position: relative;
+  align-items: center;
+  display: inline-flex;
+
+  input {
+    opacity: 0;
+    width: 0;
+    height: 0;
+  }
+
+  &__slider {
+    position: relative;
+    display: inline-block;
+    border-radius: 37px;
+    cursor: pointer;
+    width: 28px;
+    height: 18px;
+    background-color: #d7d7de;
+    transition: $slider-transition;
+    margin-right: 4px;
+
+    input:checked + & {
+      background-color: $ds-brand-medium-color;
+    }
+
+    &:before {
+      border-radius: 50%;
+      position: absolute;
+      display: inline-block;
+      content: '';
+      height: 14px;
+      width: 14px;
```

Comment:
These values are related to the slider width, right? If so, can we do something like:
```
$sliderWidth: 28px;
...
&__slider {
...
  width: @sliderWidth;
...
}
...
&:before: {
...
height: calc(@sliderWidth / 2);
width: calc(@sliderWidth / 2);
...
}

Reply:
done